composer.json

```diff
@@ -43,7 +43,7 @@
     ],
     "extra": {
         "branch-alias": {
-            "dev-master": "0.10-dev"
+            "dev-master": "1.1-dev"
```

Comment:
we did not tag 1.0.0 - i would only change this once we add a new feature to discovery.